src/image/morphology/blackHat.js

```diff
@@ -0,0 +1,19 @@
+/**
+ * @memberof Image
+ * @instance
+ * @param {Matrix} kernel
+ * @return {Image}
+ */
+export default function morphologicalGradient(kernel) {
```

Comment:
Why is the name of the method different of the name of the file ? Seems a copy / paste mistake